src/deposit/dto/update-deposit.dto.ts

```diff
@@ -1,4 +1,9 @@
 import { PartialType } from '@nestjs/mapped-types';
 import { CreateDepositDto } from './create-deposit.dto';
 
-export class UpdateDepositDto extends PartialType(CreateDepositDto) {}
+export class UpdateDepositDto extends PartialType(CreateDepositDto) {
+  id: string;
+  idUser: string;
```

Comment:
popraw na userId

Reply:
jak byśmy chcieli być dokładni to powinno być samo user